tests/jax/updating/terminators_test_data.py

```diff
@@ -0,0 +1,38 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, List, Tuple
+
+
+def count_condition_terminator_data() -> List[Dict]:
+    """Data for count condition terminator"""
+
+    return [
+        {"trainer_steps": 10},
+        {"trainer_walltime": 10},
+        {"evaluator_steps": 10},
+        {"evaluator_episodes": 10},
+        {"executor_episodes": 10},
+        {"executor_steps": 10},
+    ]
```

Comment:
This is much cleaner than what we were discussing yesterday

Reply:
:rocket: